osbs/http.py

```diff
@@ -165,7 +165,9 @@ def iter_lines(self):
             kwargs['chunk_size'] = 1
         try:
             return self.req.iter_lines(**kwargs)
-        except (requests.exceptions.ChunkedEncodingError, httplib.IncompleteRead):
+        except (requests.exceptions.ChunkedEncodingError,
```

Comment:
It probably won't be extra, yes, although I'm not sure if `ConnectionError` is possible during `iter_lines` - it would most likely be wrapped in `ChunkedEncodingError`